src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -164,6 +260,38 @@ public IAnyCallConfigurationWithReturnTypeSpecified<TMember> Where(Func<IFakeObj
                 this.ParentConfiguration.RuleBeingBuilt.ApplyWherePredicate(predicate, descriptionWriter);
                 return this;
             }
+
+            public virtual IAfterCallSpecifiedConfiguration<IReturnValueConfiguration<TMember>> AssignsOutAndRefParametersLazily(Func<IFakeObjectCall, ICollection<object>> valueProducer)

```

Comment:
Good point. I must have copied it from the enclosing class, which was virtual. I'll remove it.